components/tabs/Cargo.toml

```diff
@@ -33,14 +33,14 @@ sql-support = { path = "../support/sql" }
 sync-guid = { path = "../support/guid", features = ["random"] }
 sync15 = { path = "../sync15", features = ["sync-engine"] }
 thiserror = "1.0"
-uniffi = ">=0.19.6, <=0.20" # m-c is on 19.6, a-s is on 0.20.0
-uniffi_macros = ">=0.19.6, <=0.20" # m-c is on 19.6, a-s is on 0.20.0
+uniffi = ">=0.19.6, <=0.21" # m-c is on 19.6, a-s is on 0.21.0
```

Comment:
Note that m-c is now on [0.20.0](https://searchfox.org/mozilla-central/source/Cargo.toml#177)